var/spack/repos/builtin/packages/fplo/package.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import glob
+import os
+import platform
+
+from spack.package import *
+
+
+class Fplo(MakefilePackage):
+    """The FPLO(R) package is a full-potential local-orbital code to solve the
+    Kohn-Sham equations on a regular lattice or with free boundary conditions
+    (finite systems). Relativistic effects are treated either in a
+    scalar-relativistic or a full  4-component formalism.  Available
+    functionals are LSDA, GGA (PBE 96) and LSDA/GGA+U. Orbital polarization
+    correction can be applied."""
+
+    homepage = "https://www.fplo.de/"
+    url = "file://{0}/FPLO22.00-62.tar.gz".format(os.getcwd())
+    manual_download = True
+
+    maintainers = ["glennpj"]
+
+    version("22.00-62", sha256="0d1d4e9c1e8e41900901e26c3cd08ee39dcfdeb3f2c4c8862055eaf704b6d69e")
+
+    patch("lapackconfig.patch")
+    patch("MMakefile.patch")
+    patch("ncurses.patch")
+    patch("qt-make.patch")
+    patch("fedit_py.patch")
```

Comment:
Can you add a brief comment above each `patch` directive to briefly state provenance, aim of the patch etc. ?

Reply:
Sure, I just made a commit for this.